source/gfoidl.Base64/InternalsVisibleTo.cs

```diff
@@ -1,5 +1,10 @@
 ﻿using System.Runtime.CompilerServices;
 
-[assembly: InternalsVisibleTo("gfoidl.Base64.Benchmarks")]
-[assembly: InternalsVisibleTo("gfoidl.Base64.FuzzTests")]
-[assembly: InternalsVisibleTo("gfoidl.Base64.Tests")]
+[assembly: InternalsVisibleTo("gfoidl.Base64.Benchmarks, Publickey=" + Keys.SignKey)]
+[assembly: InternalsVisibleTo("gfoidl.Base64.FuzzTests, Publickey="  + Keys.SignKey)]
+[assembly: InternalsVisibleTo("gfoidl.Base64.Tests, Publickey="      + Keys.SignKey)]
```

Comment:
Could be simplified by https://github.com/dotnet/sdk/pull/3439 and written in the csproj.